src/views/Books/BookList.js

```diff
@@ -18,14 +19,19 @@ function BookList() {
   if (loading) return <h1>Loading books...</h1>;
 
   return (
-    <ul className="book-list" aria-label="book list">
-      {books.map((book) => (
-        <li key={book.book_id}>
-          <Book book={book} />
-        </li>
-      ))}
-    </ul>
+    <>
+      <ul className="book-list" aria-label="book list">
+        {books.map((book) => (
+          <Link key={book.book_id} to={`./books/${book.book_id}`} >
```

Comment:
We can remove the curly braces and . around the to=. We only want /path/:path in our to=